python/pyarrow/plasma.pyx

```diff
@@ -588,3 +606,66 @@ def connect(store_socket_name, manager_socket_name, int release_delay,
                               result.manager_socket_name,
                               release_delay, num_retries))
     return result
+
+def put(PlasmaClient client, list value, ObjectID object_id=None):
```

Comment:
Instead of forcing the user to pass in a list, why not just allow them to pass in anything and then wrap it in a list?